src/registrar/tests/test_dns_account.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from django.db import IntegrityError, transaction
+from django.apps import apps
+
+DnsAccount = apps.get_model("registrar", "DnsAccount")
+
+class DnsAccountTest(TestCase):
+    """
+    Testing constraints on DnsAccount model
```

Comment:
I'd remove this comment or remove "constraints on" since there will be other tests in this file that aren't related to constraints (ex. if we did a custom save() method on the model or something like that)